.azure-pipelines/win32-conda-CI.yml

```diff
@@ -112,16 +112,6 @@ jobs:
         onnxrt.version: 'onnxruntime==1.7.0'  # -i https://test.pypi.org/simple/ ort-nightly'
         onnxcc.version: 'onnxconverter-common==1.8.1'  # git+https://github.com/microsoft/onnxconverter-common.git
         sklearn.version: '==0.24.2'
-      Py38-Onnx181-Rt170-Skl0241:
-        python.version: '3.8'
```

Comment:
Python 3.8 breaks?

Reply:
yes. the log is [here](https://dev.azure.com/onnxmltools/bafa8fda-79b9-45f5-8ee6-97f3d532fc46/_apis/build/builds/22385/logs/263).
it says cannot import apply_reducesum. I don't know how to fix it.

Reply:
You should update onnxconverter-common==1.7.0 to onnxconverter-common==1.8.1, that should fix it.

Reply:
thanks @xadupre for the hint. it works.